invokeai/frontend/web/src/features/nodes/util/parseSchema.ts

```diff
@@ -13,7 +13,7 @@ import {
   buildOutputFieldTemplates,
 } from './fieldTemplateBuilders';
 
-const invocationBlacklist = ['Graph', 'Collect', 'LoadImage'];
+const invocationBlacklist = ['Graph', 'Collect'];
```

Comment:
Prefer "DenyList" for inclusivity.

Reply:
This will need more changes than just in this file. I'll tag @psychedelicious so he can fully replace them across the board.

Reply:
Oof, good call